### DISCUSSION TITLE
Accelerometer key mapping

### DISCUSSION THREAD

**Discussion by Ahmed-M0rsy** [PlayCover/PlayCover#594] [Ideas]
What about mapping **accelerometer** to the PS4 controller **sixaxis** sensor or **thumbstick**
I believe it'll make a game like **CarX Street** much better

### Finally I'd like to thank every developer and everyone collaborated on PlayCover

**Comment by amirsaam** [comment]
This kind of changes are out of PlayCover perspective also it needs to PlayCover to interfere with things that are not in its power to do.
